source/docs/0.13/moleculer-web.md

```diff
@@ -778,7 +778,7 @@ settings: {
                 "POST divide": "math.div",
                 "GET greeter/:name": "test.greeter",
                 "GET /": "test.hello",
-                "POST upload"(req, res) {
+                "POST upload": (req, res) {
```

Comment:
This syntax is not correct, but original is. Please leave the original

Reply:
True, my fault :)